casepro/backend/junebug.py

```diff
@@ -219,6 +249,9 @@ def send_message(self, message):
             }
             self.session.post(self.url, json=data)
 
+            if settings.JUNEBUG_MUST_SEND_HUB_OUTGOING_MESSAGE:
```

Comment:
You could make this `if settings.JUNEBUG_HUB_BASE_URL and settings.JUNEBUG_HUB_AUTH_TOKEN` and avoid adding a third setting?